### PR TITLE
New weekly 50 bv

### PR DESCRIPTION
**Pull request by maximenoel8**
Add the new weekly 5.0 build validation pipeline.
The purpose of this pipeline is to be run weekly and use for development.

### CODE REVIEW

File: jenkins_pipelines/environments/manager-qe-build-validation-continuous-NUE [hunk 1]
```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env groovy
```
Comment (srbarrios) [inline jenkins_pipelines/environments/manager-qe-build-validation-continuous-NUE:1]: Please let's rename this file as continuous build validation

File: terracumber_config/tf_files/SUSEManager-build-validation-continuous-NUE.tf [hunk 2]
```diff
@@ -0,0 +1,1276 @@
+// Mandatory variables for terracumber
```
Comment (srbarrios) [inline terracumber_config/tf_files/SUSEManager-build-validation-continuous-NUE.tf:1]: Let's rename this as continuous build validation please. So, it really breaks the regex as a different kind of entity.

File: terracumber_config/tf_files/SUSEManager-build-validation-continuous-NUE.tf [hunk 3]
```diff
@@ -0,0 +1,1276 @@
...
+}
+
+variable "MAIL_FROM" {
+  type = string
+  default = "galaxy-noise@suse.de"
+}
+
+variable "MAIL_TO" {
+  type = string
+  default = "galaxy-noise@suse.de"
```
Comment (srbarrios) [inline terracumber_config/tf_files/SUSEManager-build-validation-continuous-NUE.tf:54]: I'd send this to galaxy-ci or even we can create a new one for galaxy-bv@suse.de.
Make sense?

File: terracumber_config/tf_files/SUSEManager-build-validation-continuous-NUE.tf [hunk 4]
```diff
@@ -0,0 +1,1276 @@
...
+  type = string
+  default = null // Not needed for master, as it is public
+}
+
+variable "GIT_PASSWORD" {
+  type = string
+  default = null // Not needed for master, as it is public
+}
+
+variable "ZVM_ADMIN_TOKEN" {
```
Comment (srbarrios) [inline terracumber_config/tf_files/SUSEManager-build-validation-continuous-NUE.tf:80]: For the Continuous BV, I would go lighter, at least for a while, and comment or remove other archs resources.

File: terracumber_config/tf_files/SUSEManager-build-validation-continuous-NUE.tf [hunk 5]
```diff
@@ -0,0 +1,1276 @@
...
+  local_user  = "jenkins@jenkins-worker.mgr.suse.de"
+}
+
+module "base_core" {
+  source            = "./modules/base"
+
+  cc_username       = var.SCC_USER
+  cc_password       = var.SCC_PASSWORD
+  product_version   = var.PRODUCT_VERSION
+  name_prefix       = "suma-bv-continuous-"
```
Comment (srbarrios) [inline terracumber_config/tf_files/SUSEManager-build-validation-continuous-NUE.tf:119]: suma-continuous-bv- (in the future mlm-continuous-bv)

File: terracumber_config/tf_files/SUSEManager-build-validation-continuous-NUE.tf [hunk 6]
```diff
@@ -0,0 +1,1276 @@
...
+  providers = {
+    libvirt = libvirt.suma-arm
+  }
+  source             = "./modules/sshminion"
+  base_configuration = module.base_arm.configuration
+  name               = "opensuse155arm-sshminion-nue"
+  image              = "opensuse155armo"
+  provider_settings = {
+    mac                = "aa:b2:93:01:02:c7"
+    overwrite_fqdn     = "suma-bv-weekly-50-opensuse155arm-sshminion.mgr.suse.de"
```
Comment (srbarrios) [inline terracumber_config/tf_files/SUSEManager-build-validation-continuous-NUE.tf:922]: I guess those domains will change too?

File: terracumber_config/tf_files/SUSEManager-build-validation-continuous-NUE.tf (hunk 4 above)
Comment (maximenoel8) [reply terracumber_config/tf_files/SUSEManager-build-validation-continuous-NUE.tf]: Normally we have enough resources to run 3 BVs in parallel (I had 4.3, 5.0 and uyuni running together). We dropped uyuni BV pipeline so I think we have room for continuous. If we want to disable those clients, we can do it at the minionList level.

**Review by srbarrios — APPROVED** [review]

File: terracumber_config/tf_files/SUSEManager-build-validation-continuous-NUE.tf (hunk 4 above)
Comment (srbarrios) [reply terracumber_config/tf_files/SUSEManager-build-validation-continuous-NUE.tf]: OK. It was more about reducing coverage to review less, until all is stable. But ok, let's try with the whole BV.

File: terracumber_config/tf_files/SUSEManager-continuous-build-validation-NUE.tf [hunk 7]
```diff
@@ -0,0 +1,1282 @@
...
+}
+
+variable "MAIL_FROM" {
+  type = string
+  default = "galaxy-noise@suse.de"
+}
+
+variable "MAIL_TO" {
+  type = string
+  default = "galaxy-noise@suse.de"
```
Comment (srbarrios) [inline terracumber_config/tf_files/SUSEManager-continuous-build-validation-NUE.tf:60]: For some reason the previous comment gonne, I suggest we use something different than noise for that one. Either galaxy-ci, either I can create a new one for galaxy-bv

**Pull request merged**